indico/htdocs/js/indico/RegistrationForm/tpls/fields/number.tpl.html

```diff
@@ -14,7 +14,7 @@
         {{ 'Field is mandatory' | i18n }}
     </span>
     <span ng-show="validationStarted && nestedForm.$invalid && nestedForm.$error.min " class="error-message-tag">
-        {{ 'The value must be at least' | i18n }} {{field.values.minValue}}
+        {{ 'The value must be at least' | i18n }} {{field.minValue}}
```

Comment:
So I guess `{ 'The value must be at least {0}' | i18n(field.minValue) }}` could work with this modificaton:

```
ndFilters.filter("i18n", function() {
    return function(input) {
        var str = $T.gettext(input);
        if (arguments.length > 1) {
            str = str.format.apply(str, [].slice.call(arguments, 1))
        }
        return str;
    };
});
```